packages/frontend-2/components/model/DashboardList.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <section>
+    <div class="flex items-center justify-between gap-4 mt-8 mb-4">
+      <div class="flex items-center gap-4">
+        <h1 class="h4 text-foreground font-bold flex items-center">
+          {{ title }}
+          <span class="ml-2 caption">{{ numModels }}</span>
+        </h1>
+      </div>
+      <div v-if="enableSearch">
+        <FormTextInput
+          name=""
```

Comment:
`name` is required for a reason, its accessible labels rely on it (even if there's no visible label, we should support them for screen readers)

Reply:
legit, i'll re-add it